### ISSUE TITLE
Not working for Slack 3.2.0 64-bit on Windows 10

### ISSUE THREAD

**Issue by aaoferreira** [IvyBits/old-slack-emojis#6]
Hey, it seems it's not working anymore on my Slack 3.2.0 64-bit for Windows 10. Can anyone confirm it works or is it for everybody?

**Comment by jamesstanleystewart** [comment]
Can confirm. My emojis have reset and it no longer seems to work

**Comment by FrostTheFox** [comment]
Can confirm, not working here either, Windows 10 Enterprise, Slack 3.2.0 64bit.

**Comment by kroshner** [comment]
Same here, Slack 3.2.0 64-bit, Windows 10 Pro x64

**Comment by joan-s-molas** [comment]
same here. hi @aaoferreira

**Comment by zipityzi** [comment]
Alas, also experiencing this issue on 3.2.0 64-bit on Win 10 Pro x64.

**Comment by FloEberle** [comment]
to get it working again (works on my machine :trollface: ):

- run the bat file
- open %UserProfile%\Local\slack\app-3.2.0\resources\app.asar.unpacked\src\static\ssb-interop.js
- scroll to the end of the file and move the following block from outside to inside of the curly brackets, right after the init() call
```
   // ** old-slack-emojis ** https://github.com/IvyBits/old-slack-emojis
var scriptPath = path.join(__dirname, 'old-slack-emojis.js').replace('app.asar', 'app.asar.unpacked');
require('fs').readFile(scriptPath, 'utf8', (e, r) => { if (e) { throw e; } else { eval(r); } });
```

![image](https://user-images.githubusercontent.com/412740/44463610-97bd6d80-a618-11e8-8c23-cf09e5b7b299.png)

- profit

**Comment by quantum5** [comment]
I released a new version in 456264166a8196bd9e7c17225d44b01c9a39711a that should fix this issue. I thank @FloEberle for the hack that inspired the actual fix.

**Comment by FloEberle** [comment]
thanks @quantum5 😎
